app/src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java

```diff
@@ -125,19 +126,28 @@ public JsonArray getDownloads(String experimentAccession, String accessKey) {
                 ImmutableList.of(
                         ExperimentFileType.EXPERIMENT_METADATA,
                         ExperimentFileType.EXPERIMENT_DESIGN);
-
-        var resultFiles = isSmartExperiment(technologyType) ?
-                ImmutableList.of(
-                        ExperimentFileType.CLUSTERING,
-                        ExperimentFileType.QUANTIFICATION_FILTERED,
-                        ExperimentFileType.MARKER_GENES,
-                        ExperimentFileType.NORMALISED,
-                        ExperimentFileType.QUANTIFICATION_RAW) :
-                ImmutableList.of(
-                        ExperimentFileType.CLUSTERING,
-                        ExperimentFileType.MARKER_GENES,
-                        ExperimentFileType.NORMALISED,
-                        ExperimentFileType.QUANTIFICATION_RAW);
+        List<ExperimentFileType> resultFiles = new ArrayList<>();
+
+        if (experimentAccession.startsWith("E-ANND-")) {
```

Comment:
Tiny, but cleaner code:

```suggestion
        if (isAnndataExperiment(experimentAccession)) {
```

and extract the `experimentAccession.startsWith("E-ANND-")` to a private method.